node/lib/DeviceAccessor.ts

```diff
@@ -1,18 +1,107 @@
 
 import { DeviceInterface } from "./DeviceInterface";
-import { IDevice } from "./ITranslator";
+import { ITranslator } from "./ITranslator";
 
 import { EventEmitter } from "events";
 
 /**
  * Provides reflection-style access to device properties and methods via device interfaces.
  */
 export class DeviceAccessor {
```

Comment:
Yes! (But there will be no projections for specific translator interfaces, because those are dynamically derived from schema files.)

---
In reply to: [73393802](https://github.com/opent2t/opent2t/pull/2#discussion_r73393802) [](ancestors = 73393802)